### ISSUE TITLE
Loss and Acc on VGGVox (Thin-Resnet & NetVLAD)

### ISSUE THREAD

**Issue by jjeremy40** [a-nagrani/VGGVox#19]
Hi, correct me if I'm wrong but you trained VGGVox on VoxCeleb2 as Classification, right ? If so, what kind of Loss and Acc did you get ? for training set and validation set ?
Thanks.
